yarn.lock

```diff
@@ -3374,7 +3374,6 @@ __metadata:
     "@types/jest": "npm:^27.4.1"
     "@types/punycode": "npm:^2.1.0"
     deepmerge: "npm:^4.2.2"
-    eth-phishing-detect: "npm:^1.2.0"
```

Comment:
This is still in the tree due to this: https://github.com/MetaMask/metamask-mobile/pull/11111#discussion_r1752309640

Reply:
Still true today? I guess it can be removed in a follow-up in any case.